charts/yourls/templates/_helpers.tpl

```diff
@@ -45,65 +45,65 @@ Return the proper Docker Image Registry Secret Names
 Return the MySQL Hostname
 */}}
 {{- define "yourls.databaseHost" -}}
-{{- if .Values.mysql.enabled }}
-    {{- if eq .Values.mysql.architecture "replication" }}
-        {{- printf "%s-primary" (include "yourls.mysql.fullname" .) | trunc 63 | trimSuffix "-" -}}
+    {{- if .Values.mysql.enabled }}
```

Comment:
By convention, the block opened by a `define` keeps the same indentation.
This is the same for the whole file.

Reply:
The `{{-` Jinja notation strips all left-hand white spacing. Just as `-}}` strips right-hand white spacing